retrier/classifier_test.go

```diff
@@ -34,7 +34,7 @@ type wrappedErr struct {
 }
 
 func (w wrappedErr) Error() string {
-	return "there's an error happening during X: " + w.Error()
+	return "there's an error happening during X: " + w.error.Error()
```

Comment:
:thinking: not technically related related to the rest of the PR, but nice catch

I guess this just never worked, but also was never called?

Reply:
Yes, I think so